Makefile

```diff
@@ -21,7 +21,6 @@ clean:
 	rm -f VERSION
 	rm -rf vendor
 	rm -rf node_modules
-	rm package-lock.json
```

Comment:
Comparing this to main this shouldn't be removed but `rm -rf modules/electrophysiology_browser/jsx/react-series-data-viewer/node_modules` should be.

Reply:
sorry, misread the diff.. removing this line is correct but `rm -rf modules/electrophysiology_browser/jsx/react-series-data-viewer/node_modules` is missing.

Reply:
Done!